server/schedulers/utils.go

```diff
@@ -25,8 +25,9 @@ import (
 
 const (
 	// adjustRatio is used to adjust TolerantSizeRatio according to region count.
-	adjustRatio          float64 = 0.005
-	minTolerantSizeRatio float64 = 1.0
+	adjustRatio             float64 = 0.005
+	leaderTolerantSizeRatio float64 = 5.0
```

Comment:
Does this need to be configurable? /cc @nolouch

Reply:
I think it's ok now.